flink-table-store-spark/src/test/java/org/apache/flink/table/store/spark/SparkReadTestBase.java

```diff
@@ -129,12 +129,17 @@ public static void startMetastoreAndSpark() throws Exception {
     }
 
     protected static SimpleTableTestHelper createTestHelper(Path tablePath) throws Exception {
-        RowType rowType =
-                new RowType(
-                        Arrays.asList(
-                                new RowType.RowField("a", new IntType(false)),
-                                new RowType.RowField("b", new BigIntType()),
-                                new RowType.RowField("c", new VarCharType())));
+        return createTestHelper(
+                tablePath,
+                Arrays.asList(
+                        new RowType.RowField("a", new IntType(false)),
+                        new RowType.RowField("b", new BigIntType()),
+                        new RowType.RowField("c", new VarCharType())));
+    }
+
+    protected static SimpleTableTestHelper createTestHelper(
```

Comment:
Can we stop using this class? Because Spark SQL can create tables, Spark SQL can also write data.

Reply:
👍 I have created an issue to remove `SimpleTableTestHelper` in SparkITCase https://issues.apache.org/jira/browse/FLINK-30701